src/components/movie-view/movie-view.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+export class MovieView extends React.Component {
+    constructor() {
+        super();
+
+        this.state = {};
+    }
+
+
+
+    render() {
+        let { movie, onClick } = this.props;
+
+        if (!movie) return null;

```

Comment:
You can update this to be `<div>Loading...</div>` if you want to let a user know that the movie is loading